runtime/oti/j9nonbuilder.h

```diff
@@ -3764,6 +3764,7 @@ typedef struct J9JITConfig {
 	int32_t (*startJITServer)(struct J9JITConfig *jitConfig);
 	int32_t (*waitJITServerTermination)(struct J9JITConfig *jitConfig);
 #endif /* JITSERVER_SUPPORT */
+	BOOLEAN bPrintCodeCache;
```

Comment:
Unless the VM needs to know about this, I would put this in another place.
Possible alternatives:
1. privateJITConfig
2. persistentInfo
3. JIT Options
Conceptually, -XX:+PrintCodeCache being an option, maybe creating a shadow JIT option (a bit in TR::Options) makes most sense here